www/js/control/PopOpCode.jsx

```diff
@@ -1,30 +1,48 @@
-import React from "react";
+import React,  { useState } from "react";
 import { Modal, StyleSheet } from 'react-native';
 import { Button, Text, IconButton, Dialog, useTheme } from 'react-native-paper';
 import { useTranslation } from "react-i18next";
 import QrCode from "../components/QrCode";
+import AlertBar from "./AlertBar";
 
 const PopOpCode = ({visibilityValue, tokenURL, action, setVis}) => {
     const { t } = useTranslation(); 
     const { colors } = useTheme();
+
+    const opcodeList = tokenURL.split("=");
+    const opcode = opcodeList[opcodeList.length - 1];
+    
+    const [copyAlertVis, setCopyAlertVis] = useState(false);
+
+    const copyText = function(textToCopy){
+        navigator.clipboard.writeText(textToCopy).then(() => {
```

Comment:
Is this still iOS-only ? Last I checked we couldn't copy text on Android without extra permissions.
What happens when Android users click the copy button? Nothing?

Reply:
You're right, I forgot about that. I just changed it so that nothing happens on Android, though now that I think of it, it would be better for the text to be clickable, and to maybe not have a copy button since a button that does nothing is confusing.

Reply:
Or just only show the button on iOS...

Reply:
that was a better idea, made that change